### PR TITLE
feat(frontend): add USDT token and link it to ckUSDT

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We add USDT as token in the env folders, and we link it to the new ckUSDT token. Furthermore, we remove the "temporary" USDT token that was there.

### CODE REVIEW

File: src/frontend/src/eth/utils/erc20.utils.ts [hunk 1]
```diff
@@ -64,8 +63,6 @@ const mapErc20Icon = (symbol: string): string | undefined => {
 			return rndr;
 		case 'uni':
 			return uniswap;
-		case 'usdt':
```
Comment (peterpeterparker) [inline src/frontend/src/eth/utils/erc20.utils.ts:67]: redo?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/eth/utils/erc20.utils.ts:67]: we don't use that one for USDT anymore, correct? now the icon is included in the `env` file

Comment (peterpeterparker) [reply src/frontend/src/eth/utils/erc20.utils.ts:67]: Ah yeah! thanks

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

**Pull request merged**